sources/api/certdog/src/main.rs

```diff
@@ -36,9 +39,9 @@ struct Args {
     #[argh(option, default = "LevelFilter::Info", short = 'l')]
     /// log-level trace|debug|info|warn|error
     log_level: LevelFilter,
-    #[argh(option, default = "constants::API_SOCKET.to_string()", short = 's')]
-    /// socket-path path to apiserver socket
-    socket_path: String,
+    #[argh(option, default = "DEFAULT_CONFIG_FILE.to_string()", short = 'c')]
```

Comment:
This looks weird. I guess this is correctly resolving to the constant+function call and not setting the default to the string literal `DEFAULT_CONFIG_FILE.to_string()`? Did you check this? I don't know if the Bottlerocket call sites are specifying the path or relying on this default.